packages/storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,100 @@
+<style>
+    /* the !importants are a result of computed selectors - will need to re-visit */
+    /* the computed selectors come from something to do with story book and their style sheets */
+    /* Escape hatches only seem to work on selectors that have story book formatting - .sbdocs.sbdocs-h1 for example */
+    /* !important is used to force some of the styles that go onto elements that do not get these special selectors */
+
+    .sbdocs.sbdocs-h1,
+    .sbdocs.sbdocs-h2,
+    .sbdocs.sbdocs-h3,
+    .sbdocs.sbdocs-p {
+        font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        color: #000000;
+    }
+
+    .sbdocs.sbdocs-h1,
+    [class*="sbdocs-title css-"],
+    .sbdocs.sbdocs-h1:first-of-type {
+        color: #000000;
+        font-size: 40px;
```

Comment:
avoid using 'px', instead use 'rem' (with 1rem being 16px by default)
we can use 'em' also but we have to be aware of the nesting using 'em' implies :)

Reply:
ah okay, I was using the styles Alex P sent over, will do the math to make that work!